tests/trainer/logging/test_logger_connector.py

```diff
@@ -425,3 +425,21 @@ def test_dataloader(self):
     )
     trainer.fit(model)
     trainer.test(model, ckpt_path=None)
+
+
+def test_logging_to_progress_bar_with_reserved_key(tmpdir):
+    """ Test that logging a metric with a reserved name to the progress bar raises a warning. """
+    class TestModel(BoringModel):
+
+        def training_step(self, *args, **kwargs):
+            output = super().training_step(*args, **kwargs)
+            self.log("loss", output["loss"], prog_bar=True)
+            return output
+
+    model = TestModel()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_steps=2,
+    )
+    with pytest.warns(UserWarning, match="The progress bar already tracks a metric with the name 'loss'"):
```

Comment:
```suggestion
    with pytest.warns(UserWarning, match="The progress bar already tracks a metric with the name(s) 'loss'"):
```

Reply:
the parentheses will not work, this is a regular expression
needs to be a raw string I think